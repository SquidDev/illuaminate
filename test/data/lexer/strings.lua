"a"
"a\nb"
"a\0b"
"a\x00b"
"a\(b"

'a'
'a\nb'
'a\0b'
'a\x00b'
'a\(b'

"\u{1F991}"
