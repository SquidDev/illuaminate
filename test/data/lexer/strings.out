"a"
(Node.Whitespace "\n")
"a\nb"
(Node.Whitespace "\n")
"a\0b"
(Node.Whitespace "\n")
"a\x00b"
(Node.Whitespace "\n")
"a\(b"
(Node.Whitespace "\n")
(Node.Whitespace "\n")
'a'
(Node.Whitespace "\n")
'a\nb'
(Node.Whitespace "\n")
'a\0b'
(Node.Whitespace "\n")
'a\x00b'
(Node.Whitespace "\n")
'a\(b'
(Node.Whitespace "\n")
(Node.Whitespace "\n")
"\u{1F991}"
(Node.Whitespace "\n")
end of file
